src/io/flutter/project/FlutterProjectOpenProcessor.java

```diff
@@ -103,7 +104,14 @@ else if (modules.size() > 1) {
           LOG.warn(MessageFormat.format("{0} contains {1} modules.", project.getName(), modules.size()));
```

Comment:
I *believe* this will just get logged and so should go pretty much unnoticed unless folks are seeing issues in which case maybe it's useful?  That said, I'm up for a re-think.